.github/workflows/ci.yml

```diff
@@ -49,11 +49,14 @@ jobs:
     if: github.event_name == 'schedule' || github.event_name == 'push' || github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id
     strategy:
       matrix:
-        os: [ubuntu-20.04, macos-11, windows-2022]
+        os: [ubuntu-22.04, macos-11, windows-2022]
```

Comment:
I think we should stick with ubuntu-20.04 here for now so that the Arm native library is compatible with ubuntu 20.04, meaning we can remove the extra include below for arm64 on ubuntu-20.04.

It doesn't make sense to have both ubuntu-22.04 arm64 and ubuntu-20.04 arm64, as these will both have the same value for `$triplet` so be trying to create the same artifact name.